src/anndata2ri/scipy2ri/py2r.py

```diff
@@ -27,19 +27,26 @@ def get_type_conv(dtype: np.dtype) -> Tuple[str, Callable[[np.ndarray], Sexp], T
         raise ValueError(f'Unknown dtype {dtype!r} cannot be converted to ?gRMatrix.')
 
 
+def _py2r(x):
+    import rpy2.robjects as ro
+
+    with localconverter(default_converter + numpy2ri.converter):
+        x = ro.conversion.py2rpy(x)
```

Comment:
This isn't working. Since we know a indices, indptr and data are always `np.ndarray`s, could we just use `numpy2ri.py2rpy(x)`?

Reply:
Activating a converter means that the corresponding `py2rpy` function will be used, so doing that will change nothing.